packages/esm-ward-app/src/ward-workspace/admit-patient-form-workspace/admit-patient-form.workspace.tsx

```diff
@@ -25,11 +23,11 @@ const AdmitPatientFormWorkspace: React.FC<AdmitPatientFormWorkspaceProps> = ({
   const { location } = useWardLocation();
   const { currentProvider } = useSession();
   const [isSubmitting, setIsSubmitting] = useState(false);
-  const { mutate: mutateInpatientRequest } = useInpatientRequest();
   const { emrConfiguration, isLoadingEmrConfiguration, errorFetchingEmrConfiguration } = useEmrConfiguration();
   const [showErrorNotifications, setShowErrorNotifications] = useState(false);
   const wardPatientGrouping = useAppContext<WardPatientGroupDetails>('ward-patients-group');
   const { isLoading, mutate: mutateAdmissionLocation } = wardPatientGrouping?.admissionLocationResponse ?? {};
+  const { mutate: mutateInpatientRequest } = wardPatientGrouping?.inpatientRequestResponse ?? {};
```

Comment:
Is this being used anywhere?

Reply:
yeah, I refactored this to use the `wardPatientGroping` context instead of calling `useInpatientRequest()`.